### DISCUSSION TITLE
Can we add custom scope to newly added client

### DISCUSSION THREAD

**Discussion by dhivah** [fief-dev/fief#186] [Q&A]
It would be great if there were more features around API Client Oauth authentication. I am looking for a feature where we could add custom scopes and claims while creating a new Client credential. I saw there is an existing ticket in the project plan.

https://github.com/orgs/fief-dev/projects/1/views/4?filterQuery=custom&pane=issue&itemId=2760582

Is this feature on the roadmap in the near future?

**Comment by frankie567** [comment]
That's indeed in the roadmap, but I don't have any ETA for the moment 🙂
